index.html

```diff
@@ -437,77 +417,58 @@ <h2 id="3Definitions">Definitions</h2>
 2<sup>8</sup>-1.</dd>
 
 <!-- Maintain a fragment named "3byteOrder" to preserve incoming links to it -->
-<dfn id="3byteOrder">
-<dt>byte order</dt></dfn>
+<dt id="3byteOrder"><dfn>byte order</dfn></dt>
 
-<dd>ordering of <span class=
-"Definition"><a>bytes</a></span> for multi-byte data values within a
-<span class="Definition"><a>PNG file</a></span>
-or <span class="Definition"><a href="#4Concepts.Format"></a>
-</span>. PNG uses
-<span class="Definition"><a>network byte
-		order</a></span>.</dd>
+<dd>ordering of <a>bytes</a> for multi-byte data values within a
+<a>PNG file</a>
+or <a>PNG datastream</a>. PNG uses
+<a>network byte order</a>.</dd>
 
-<dt>
-<dt>canvas</dt></dfn>
+<dt><dfn>canvas</dfn></dt>
 
 <dd>the area on the output device on which the frames are to be displayed.
   The contents of the canvas are not necessarily available to the decoder.
   If a <span class="chunk">bKGD</span> chunk exists,
   it may be used to fill the canvas if there is no preferable background</dd>
 
 <!-- Maintain a fragment named "3channel" to preserve incoming links to it -->
-<dfn id="3channel">
-<dt>channel</dt></dfn>
-
-<dd>array of all per-<span class=
-"Definition"><a>pixel</a></span> information of a particular kind
-within a <span class=
-"Definition"><a>reference image</a></span>. There are five kinds of
-information: red, green, blue, <span class=
-"Definition"><a>greyscale</a></span>, and <span
-class="Definition"><a>alpha</a></span>. For example the alpha
+<dt id="3channel"><dfn>channel</dfn></dt>
+
+<dd>array of all per-<a>pixel</a> information of a particular kind
+within a <a>reference image</a>. There are five kinds of
+information: red, green, blue, <a>greyscale</a>, and <a>alpha</a>. For example the alpha
 channel is the array of alpha values within a reference
 image.</dd>
 
 <!-- Maintain a fragment named "3chromaticity" to preserve incoming links to it -->
-<dfn id="3chromaticity">
-<dt>chromaticity</dt></dfn>
+<dt id="3chromaticity"><dfn>chromaticity</dfn></dt>
 
 <dd>pair of CIE <i>x,y</i> values [[COLORIMETRY]] that precisely specify a colour,
 except for the brightness information.</dd>
 
 <!-- Maintain a fragment named "3chunk" to preserve incoming links to it -->
-<dfn id="3chunk">
-<dt>chunk</dt></dfn>
+<dt id="3chunk"><dfn>chunk</dfn></dt>
 
-<dd>section of a <span class=
-"Definition"><a href="#4Concepts.Format"></a></span>. Each chunk has a chunk
+<dd>section of a <a>PNG datastream</a>. Each chunk has a chunk
 type. Most chunks also include data. The format and meaning of
 the data within the chunk are determined by the chunk type.
 Each chunk is either a
-<span class=
-"Definition"><a>critical chunk</a></span> or an
-<span class=
-"Definition"><a>ancillary chunk</a></span>.
+<a>critical chunk</a> or an
+<a>ancillary chunk</a>.
 </dd>
 
 <!-- Maintain a fragment named "3colourType" to preserve incoming links to it -->
-<dfn id="3colourType">
-<dt>colour type</dt></dfn>
+<dt id="3colourType"><dfn>colour type</dfn></dt>
```

Comment:
In the text, `PNG image type ` seems to be used instead of `colour type`, and it is a better term because alpha is not a color.